server/resources.py

```diff
@@ -0,0 +1,56 @@
+from flask_restful import Resource, reqparse
+from models.users import UserModel
+
+# TODO: Replace with Marshmallow
+parser = reqparse.RequestParser()
+parser.add_argument('username', help = 'This field cannot be blank', required=True)
+parser.add_argument('password', help = 'This field cannot be blank', required=True)
+
+class UserRegistration(Resource):
+    def post(self):
+        data = parser.parse_args()
+        new_user = UserModel(
+            username = data['username'],
+            password = data['password']
+        )
+        try:
+            new_user.save_to_db()
+            return {
+                'message': 'User {} was created'.format(data['username'])
+            }
+        except Exception as e:
+            return {'message': 'Something went wrong'}, 500
+
+        # TODO: Need to raise 409 error if username already exists
+
+
+
+class UserLogin(Resource):
+    def post(self):
+        data = parser.parse_args()
+        return data
+
+class UserLogoutAccess(Resource):
+    def post(self):
+        return {'message': 'User logout'}
+
+class UserLogoutRefresh(Resource):
+    def post(self):
+        return {'message': 'User logout'}
+
+class TokenRefresh(Resource):
+    def post(self):
+        return {'message': 'Token refresh'}
```

Comment:
It would be awesome if we can implement all of these, just as an FYI, the bare minimum we need though is UserLogin, UserRegistration, UserLogout - don't need Refresh Tokens yet if it's too involved at the moment